docs/en_US/model_compression.rst

```diff
@@ -22,5 +22,6 @@ For details, please refer to the following tutorials:
     Automatic Model Compression <Compressor/AutoCompression>
     Model Speedup <Compressor/ModelSpeedup>
     Compression Utilities <Compressor/CompressionUtils>
+    Compression Benchmark <Compressor/Benchmark>
```

Comment:
this is not benchmark, it is benchmark result. So better to move it to `Use Cases and Solutions` under `Performance measurement, comparison and analysis`

Reply:
> this is not benchmark, it is benchmark result. So better to move it to `Use Cases and Solutions` under `Performance measurement, comparison and analysis`

Exactly, thanks